WooCommerce/Resources/Images.xcassets/icon-card.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "filename" : "icon-card-reader.pdf",
```

Comment:
nit: Should we use the same name for the imageset and filename? i.e The imageset is named as `icon-card` and the pdf filename is `icon-card-reader`.